lib/version_utils.pm

```diff
@@ -605,11 +611,18 @@ Returns 1 (true) if the ID_LIKE variable contains C<distri_name>.
 
 =cut
 sub check_os_release {
-    my ($distri_name, $line, $os_release_file) = @_;
+    my ($distri_name, $line, %args) = @_;
     die '$distri_name is not given' unless $distri_name;
     die '$line is not given'        unless $line;
-    $os_release_file //= '/etc/os-release';
-    my $os_like_name = script_output("grep -e \"^$line\\b\" ${os_release_file} | cut -d'\"' -f2");
+    my $go_to_target    = $args{target}     //= '';
```

Comment:
Introducing `$go_to_target` here is enough, then it can be called as `check_os_release('sles', 'ID', "$runtime run --rm $image")`.

Reply:
That is true, please check now.